### DISCUSSION TITLE
ArchR with Windows: Space in Username causes issues after creating arrowfiles.

### DISCUSSION THREAD

**Discussion by Tomoliverberry** [GreenleafLab/ArchR#886] [Questions / Documentation]
I managed to make arrowfiles without any issue so I am surprised that my username didn't cause an error at this step, but I am trying to use addDoubletScores:

_doubleScores <- addDoubletScores(input = Arrowfiles, k = 10, knnMethod = "UMAP", LSIMethod = 1)_

and the error is as follows:

  _Error in ArchRProject(ArrowFiles = ArrowFile, outputDirectory = tmpDir,  : 
  outputDirectory cannot have a space in the full path! Full path : C:\Users\Tom Berry\Desktop\archrwork_
  
  Not sure why it worked at one stage and not the other but does anyone have a work around to this issue aside from changing the user?

**Comment by rcorces** [comment]
Sorry - we dont provide any support for Windows so you wont hear from any devs on this.

**Comment by travilucas** [comment]
delete the space between 'Tom Berry'
